### ISSUE TITLE
[Gnome 3.30, v15] Missing Devices in settings

### ISSUE THREAD

**Issue by Znert** [andyholmes/gnome-shell-extension-gsconnect#343]
**Describe the bug**
When opening the `Mobile Settings` I cannot see devices that are currently connected.

**To Reproduce**
Unclear. I assume that the update from Gnome 3.28 to 3.30 while still on v12 caused the issue.

**Expected behavior**
The `Mobile Settings` must show all devices that are known.

**Screenshots**
![screenshot from 2018-11-19 21-13-04](https://user-images.githubusercontent.com/18900196/48733248-556cf200-ec42-11e8-9ede-e27fa834e149.png)

As you can see, my mobile phone is currently connected (and working properly regarding actions I can actually test like ping for remote input), but does not show the device or any other paired device I have in the settings.

The only device I had seen here was my old phone, but I have already removed in hopes that that would trigger a refresh of some sorts. I do not know why exactly my old phone showed up but the rest doesn't.

**Debug output**
I tried to look into the debug output, but I cannot find anything fishy, just a bunch of information about the connected device, I assume that's just (sensitive) data about the connection etc.

**System Details (please complete the following information):**
 - GSConnect version: 15
   - Installed from: extensions.gnome.org
 - GNOME/Shell version: 3.30
 - Distro/Release: Manjaro

**GSConnect environment (if applicable):**
 - Paired Device(s): Nokia 7 Plus (in screenshot), HP Spectre
 - KDE Connect app version: latest on Google Play

**Additional context**
I assume that something went wrong during updating the extension. I already tried to reinstall GSConnect, but without luck as all settings are saved somewhere else... Maybe if I could hard reset the extension?

**Comment by ferdnyc** [comment]
@Znert Yeah, sorry, v12 is definitely not 3.30 compatible, and updating from v12 all the way to v15 occasionally causes some issues. The extension underwent a _complete_ rewrite (v13), followed by significant enhancements (v15) in the interim.

If you've tried completely rebooting your system (which is sometimes all it takes), and GSConnect still isn't seeing your device, then following these steps should sort things out (though you'll have to re-pair your phone and start from scratch in terms of settings):
1. Uninstall GSConnect
2. Run [these commands to remove all settings](https://github.com/andyholmes/gnome-shell-extension-gsconnect/releases), from the wiki.
3. Install GSConnect again, either:
    * v15 from [extensions.gnome.org](https://extensions.gnome.org/extension/1319/gsconnect/)
    * v16 (the next release, not yet reviewed by gnome) [from the releases page here](https://github.com/andyholmes/gnome-shell-extension-gsconnect/releases)

If you decide to download v16, there are [instructions for installing from ZIP](https://github.com/andyholmes/gnome-shell-extension-gsconnect/releases), also on the wiki.

After that, and one last restart to be sure, you should be up and running again.

**Comment by Znert** [comment]
Thanks a lot for your swift response :)

I did as you said and reset the extension, everything seems to work now, thanks!

And sorry that I didn't find the removal guide myself.

**Comment by andyholmes** [comment]
No problems. The old Wiki was split into many pages, but Github's statistics was telling me no one was looking deeper than the FAQ, so I've been watching those and trying to organize it better.

I'll try to remember this case of that info being non-obvious ;)

**Comment by ferdnyc** [comment]
@Znert 
> And sorry that I didn't find the removal guide myself.

Not at all, and apologies if my answer came off a little "RTFM". That wasn't my intention at all (though I can see how it might've sounded that way). As @andyholmes says, things can be hard to find in the current documentation.

Truth is, I had no idea _myself_ that the removal commands were in the wiki, and I'd initially typed out a version of the same info in my response — but I happened to spot the info in the wiki while I was looking up the zip-install instructions, so I deleted that part of my response and replaced it with a link. :laughing:
